### DISCUSSION TITLE
spancat visualiser and html

### DISCUSSION THREAD

**Discussion by goonhoon** [explosion/spaCy#12454] [Help: Other Questions] [answered]
Hi,

I see that in the displaCy html the whole html gets rendered under one < div > and multiple < span > tags for each annotated span like this:

```
<span style="font-weight: bold; display: inline-block; position: relative; height: 60px;">
    BRANDING
    
<span style="background: #ddd; top: 40px; height: 4px; left: -1px; width: calc(100% + 2px); position: absolute;">
</span>

    
</span>

<span style="font-weight: bold; display: inline-block; position: relative; height: 60px;">
    AND
    
<span style="background: #ddd; top: 40px; height: 4px; left: -1px; width: calc(100% + 2px); position: absolute;">
</span>

    
</span>

<span style="font-weight: bold; display: inline-block; position: relative; height: 60px;">
    ADVERTISING
```

I am trying to make a simple Flask experiment and cannot figure out how to retain text markdown while applying my model and displaCy on the text. 

Are there any examples of customising the displaCy render? 

Many thanks,
GH

**Comment by kadarakos** [comment, marked as answer]
Hey goonhoon,

I apologize for not answering this question for such a long time. The tutorial on displaCy can be found here: https://spacy.io/usage/visualizers. Customizing the rendering is explained here: https://spacy.io/usage/visualizers#html. Let me know if you did not find the answer to your question in the tutorial.
